corehq/apps/app_manager/suite_xml/contributors/scheduler.py

```diff
@@ -0,0 +1,32 @@
+from corehq.apps.app_manager import id_strings
+from corehq.apps.app_manager.suite_xml.basic import SuiteContributor
+from corehq.apps.app_manager.suite_xml.xml import Schedule, ScheduleFixture, ScheduleVisit
+
+
+class SchedulerContributor(SuiteContributor):
+    section = 'fixtures'
+    
+    def contribute(self):
+        self.suite.fixtures.extend(self.fixtures)
+
+    @property
+    def fixtures(self):
```

Comment:
Yes that's the idea. This class has access the the Suite object so can basically add / change anything.